.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ orbs:
 executors:
   integration_test_exec: # declares a reusable executor
     docker:
-      - image: cimg/base:stable-20.04
+      - image: cimg/openjdk:17.0.4-browsers
```

Comment:
Since the java version is used in 6 places, I suggest making a parameter for it so it's easy to update the version the next time we do. An example is the `java-tag` parameter in the CircleCI config in the webservice [here](https://github.com/dockstore/dockstore/blob/afa15d20dbcc4b8479ac30d9bc8e9a219c4f37a1/.circleci/config.yml#L3-L5) and [here](https://github.com/dockstore/dockstore/blob/afa15d20dbcc4b8479ac30d9bc8e9a219c4f37a1/.circleci/config.yml#L11) is an example of how the parameter is used.